stock_picking_auto_create_lot_qty/README.rst

```diff
@@ -0,0 +1,111 @@
+======================================
+Stock Picking Auto Create Lot Quantity
+======================================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:31abf6223e281237e7c286a76e4f06a04aaeff5bb04aaef7548eee5df36c5256
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Production%2FStable-green.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Production/Stable
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fstock--logistics--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/stock-logistics-workflow/tree/16.0/stock_picking_auto_create_lot_qty
+    :alt: OCA/stock-logistics-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/stock-logistics-workflow-16-0/stock-logistics-workflow-16-0-stock_picking_auto_create_lot_qty
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/stock-logistics-workflow&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+Add a unique lot sequence for lots being created for every company
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Inventory > Master Data > Products*, select a product. #. Set
+'Auto create lot' option, and set 'Create lot every' count of UoMs for
+the products you need. #. Go the *Settings > Inventory > Stock Serial
+Lot*. #. Press --> Sequences button to create/configure your lot
+sequence for a company in 'ir.sequence' module. The default sequence is
+'stock.lot.serial'. #. Once a sequence is created/updated select the
+stock lot sequence for the current company.
```

Comment:
these still won't appear correctly on the web view, can you fix this?

Reply:
Thanks,I'll fix it